site/functions/upload.ts

```diff
@@ -9,8 +10,9 @@ export const onRequest: PagesFunction<Env> = async (context) => {
     await context.env.BUCKET.put('latest.png', stream1),
     await context.env.BUCKET.put(filename, stream2),
   ]);
-  context.waitUntil(
-    fetch('http://ollie.arjunchib.com/webhook', { method: 'POST' })
-  );
+  // assuming one for now but should eventually allow a comma-separated string
+  if (context.env.BOARD_WEBHOOKS) {
+    context.waitUntil(fetch(context.env.BOARD_WEBHOOKS, { method: 'POST' }));
+  }
```

Comment:
This sends a request to the webhook server running on the board triggering a display update. I had hardcoded your board's url here which would trigger the update from my local env or the preview env for this PR. Switching to an environment variable should fix